lib/mnesia/src/mnesia_hook.erl

```diff
@@ -74,8 +74,32 @@ do_post_commit(Tid, Commit) ->
                           , schema_ops => SchemaOps
                           },
             try Fun(Tid, CommitData)
-            catch EC:Err:Stack ->
-                    mnesia_lib:error("Mnesia post_commit hook failed: ~p:~p~nStacktrace:~p~n", [EC, Err, Stack])
+            catch EC:Err:St ->
+                    CommitTabs = commit_tabs(Ram, Disc, DiscOnly, Ext),
+                    mnesia_lib:error("Mnesia post_commit hook failed: ~p:~p~nStacktrace:~p~nCommit tables:~p~n",
```

Comment:
I think we need to change the log level to debug. If the hook needs to report some error to the operator, it can do so internally.

Reply:
Fixed